examples/python/src/authors/models.py

```diff
@@ -0,0 +1,15 @@
+# Code generated by sqlc. DO NOT EDIT.
+from typing import Optional
+
+import pydantic
+
+
+# Enums
+
+# Models
```

Comment:
We shouldn't generate this comments, they're mainly just noise.

Reply:
I agree, these are leftover from my initial single mega-file implementation, I'll remove them.